sys/include/suit/v3/suit.h

```diff
@@ -38,13 +40,13 @@ extern "C" {
  * @brief   Buffer size used for Cose
  */
 #ifndef SUIT_COSE_BUF_SIZE
-#define SUIT_COSE_BUF_SIZE              (512U)
+#define SUIT_COSE_BUF_SIZE              (180U)
 #endif
 
 /**
- * @brief   Maximum number of components used for SUIT v4
+ * @brief   Maximum number of components used for SUIT v3
```

Comment:
Here too, how about being precise in the comments/doc and specify draft-ietf-suit-manifest-03 instead of just "v3" ?

Reply:
reworded